### ISSUE TITLE
Hourly rate user

### ISSUE THREAD

**Issue by giannigio** [kevinpapst/kimai2#451]
can i calculate only hour rate for a user?
example user 1 work 3 hours and rate 20 = 60 euro
                user 2 work 2 hour and rate 10 = 20 euro
in the timesheet report the hour is correct but in the invoice calculate for working day
can you help me?
thank you

**Comment by kevinpapst** [comment]
I don't understand the question.
What exactly is wrong in the invoice?
Invoice is pretty "stupid" - it adds all selected timesheet rates, it doesn't calculate anything on its own.

**Comment by giannigio** [comment]
the timesheet it does not calculate the hourly cost but the daily rate.
user 1 works 2 hour but pay ever for 1? understand?

**Comment by giannigio** [comment]
i did a test and only calulate fixed rate. not calculate the hourly rate

**Comment by kevinpapst** [comment]
No, I don't understand. Please make an example like I did above. Also read: https://github.com/kevinpapst/kimai2/blob/master/var/docs/timesheet.md#rate-calculation

**Comment by giannigio** [comment]
Ok the example is this
User 1 works 1 hour for 50 €/h = 0 €
User 2 works 3 hour for 20 €/h = 0€
the system not calculate the hour rate field..

**Comment by kevinpapst** [comment]
Do you have any feedback on this one?

**Comment by giannigio** [comment]
It's ok!
I was wrong to insert correctly
